build.gradle

```diff
@@ -871,22 +876,30 @@ project('test:system') {
         systemProperty "imageVersion", System.getProperty("imageVersion") // pravega version.
         // bookkeeper version defaults to pravega version.
         systemProperty "pravegaBookkeeperVersion", System.getProperty("pravegaBookkeeperVersion", System.getProperty("imageVersion"))
-
+        systemProperty "bookkeeperImageVersion", System.getProperty("bookkeeperImageVersion", "latest") // // bookkeeper image version
         // tier2Type , default is NFS.
         systemProperty "tier2Type", System.getProperty("tier2Type", "nfs")
         // tier2 Configuration, specified as comma seperated key values k1=v1,k2=v2
         systemProperty "tier2Config", System.getProperty("tier2Config")
-
+        systemProperty "imageVersionPrOp",System.getProperty("imageVersionPrOp", "latest")
+        systemProperty "imageVersionBkOp",System.getProperty("imageVersionBkOp", "latest")
+        systemProperty "imageVersionZkOp",System.getProperty("imageVersionZkOp", "latest")
+        systemProperty "chartName",System.getProperty("chartName", "pravega")
+        systemProperty "repository",System.getProperty("repository", "pravega")
+
+        systemProperty "imageVersionPrOp", System.getProperty("imageVersionPrOp")
```

Comment:
Why did you do this twice (here and a few lines above)?

Reply:
Removed two entries for same parameter